.github/workflows/ci.yml

```diff
@@ -204,37 +204,41 @@ jobs:
           pip install -r tests/requirements.txt
           pip install -r requirements.txt
           python setup.py develop
-          pip install codecov-cli
+          pip install 'codecov-cli>=0.4.6'
 
       - name: Run tests
         run: |
           . venv/bin/activate
-          python -m pytest --cov=./
+          python -m pytest --cov=./ --junitxml=junit.xml
```

Comment:
Thoughts of using the workflow you did in this PR, https://github.com/codecov/gha-workflows/pull/18/files, so that it's the same across all repos and you don't have to change it in multiple places? Like here, https://github.com/codecov/worker/blob/main/.github/workflows/ci.yml#L24